test/e2e/framework/helper/certificaterequests.go

```diff
@@ -120,6 +121,27 @@ func (h *Helper) ValidateIssuedCertificateRequest(cr *v1alpha1.CertificateReques
 		expectedDNSName = expectedDNSNames[0]
 	}
 
+	usages := make(map[v1alpha1.KeyUsage]bool)
+	for _, u := range cr.Spec.Usages {
+		usages[u] = true
+	}
+	if cr.Spec.IsCA {
+		if !cert.IsCA {
+			return nil, fmt.Errorf("Expected csr cert to have IsCA set to true, but was false")
+		}
+		if cert.KeyUsage&x509.KeyUsageCertSign == 0 {
+			return nil, fmt.Errorf("Expected csr cert to have x509.KeyUsageCertSign bit set but was not")
+		}
+		usages[v1alpha1.UsageCertSign] = true
+	}
```

Comment:
Do we have a similar check for IsCA's key usages in the `ValidateIssuedCertificate` function?

Reply:
Ignore - I see you've added it 🙈